src/libraries/System.IO.IsolatedStorage/src/System.IO.IsolatedStorage.csproj

```
@@ -3,7 +3,10 @@
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
```

now that https://github.com/dotnet/runtime/pull/37944 is merged you can add an AnyOS slice:

```suggestion
    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)</TargetFrameworks>
```